src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -75,12 +127,21 @@ RouterExecutorStart(QueryDesc *queryDesc, int eflags, Task *task)
 	/* ensure that the task is not NULL */
 	Assert(task != NULL);
 
-	/* disallow transactions and triggers during distributed modify commands */
+	/* disallow triggers during distributed modify commands */
```

Comment:
Unrelated to this commit: But I doubt that SKIP_TRIGGERS has any effect here? We don't ever reach nodeModifyTuple.c, which is responsible for firing triggers...